.github/workflows/ci.yml

```diff
@@ -129,6 +129,7 @@ jobs:
       - name: Test debug
         if: matrix.kind == 'test_debug'
         run: |
+          cargo build --locked -p test_native_plugin
```

Comment:
Can this step be somehow integrated into `cli/tests/integration_tests.rs` flow?

Reply:
I tried to add a single test to `test_native_plugin`, but it still doesn't build automatically when running `cargo test --all-targets`. Not sure if there may be some other simple workaround here.

Reply:
We should figure out a way to test this without making changes to ci.yml... I'm looking into it.